text/code-standards/external-contributors.md

```diff
@@ -0,0 +1,19 @@
+# Code Review Standards for External Contributors
+
+1. __Code Line Limit__: All pull requests should not exceed 500 lines of code. This helps to ensure that changes are small and easy to review, making it less likely for bugs to be introduced and easier to understand the changes. It also keeps the pull requests manageable in size and reduces the time needed for reviewing and approving.
```

Comment:
```suggestion
1. __Code Line Limit__: Pull requests should aim to not exceed 500 lines of additions. This helps to ensure that changes are small and easy to review, making it less likely for bugs to be introduced and easier to understand the changes. It also keeps the pull requests manageable in size and reduces the time needed for reviewing and approving.
```

As much as I dislike reviewing giant PRs, sometimes they're unavoidable for refactors, especially if there's loads of tests etc. Maybe this is better?

Reply:
Yeah, that wording is nicer. Thanks!

Reply:
Could we also exclude tests from the line count? It's very easy to get to 500 lines with a few e2e tests (especially if JSON ABI files are also added).